app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -9,6 +9,7 @@ $danger: $bright-red;
 
 // Body
 $body-bg: $sul-body-bg;
+$body-color: $sul-text-color;
```

Comment:
The problem here is that `$sul-text-color` is set to `#000` which is not the color specified by the stanford identity guide and neither is it consistent with the styles on the library website.

Reply:
I'm not sure why Stanford's current branding colors matter. This was how the site was originally designed and took into account things like color contrast. It's fine if we want to adjust colors later, but getting back to the original design seems like it fixes some regressions.

Reply:
I propose we merge this change and consider a branding/palate update as a separate issue to be addressed with the help of a designer.